R/realtime.service.R

```diff
@@ -8,7 +8,7 @@
 #'@importFrom jsonlite toJSON
 #'@import XML
 #'@examples
-#'serviceEndpoint  <-	'http://olga.er.usgs.gov/data/NACCH/GOM_erosion_hazards_metadata.xml'
+#'serviceEndpoint  <-	system.file('extdata', "GOM_erosion_hazards_metadata.xml", package = 'hazardItems')
```

Comment:
seems to work ok for me locally walking through the example 
![image](https://cloud.githubusercontent.com/assets/6920344/20947555/587479f4-bbd5-11e6-92c3-fcd2a38e856a.png)